src/storage/mod.rs

```diff
@@ -61,60 +66,60 @@ pub enum Command {
         ctx: Context,
         key: Key,
         start_ts: u64,
-        callback: Callback<Option<Value>>,
+        callback: Option<Callback<Option<Value>>>,
```

Comment:
We need call cb any where occur an error or command finished, but read command and write command have different control flow, if don't wrapped by Option I will encounter some compile error. Any good suggestion?